src/wpool.erl

```diff
@@ -22,10 +22,21 @@
                   , {workers, 100}, {worker, {wpool_worker, undefined}}
                   ]).
 
--type name() :: atom().
```

Comment:
Due to the removal of this type, the app can't compile now, the compiler generates this error:
```erlang
src/wpool.erl:178: type name() undefined
```